src/consts/faq-keys-general.ts

```diff
@@ -64,7 +64,7 @@ export const faqKeysGeneral = [
   "how-sort-feed",
   "how-change-design",
   "what-is-deck",
-  "what-deck-types",
+  "what-deck-_types",
```

Comment:
This probably creates bug because this key is coming from translations. @dkildar